### ISSUE TITLE
Support for nested lists

### ISSUE THREAD

**Issue by kalyazin** [yoshiki/yaml-mode#28]
Hi,

Is the following construction going to be supported (and properly indented)?

```
title:
-
   - foo_1: bar_1
     foo_2: bar_2
-
   - foo_3: bar_3
     foo_4: bar_4```

**Comment by antalk2** [comment]
https://github.com/antalk2/yaml-mode/commit/e2e3a257c7239f0570f8fde816a7d1320afa1daf
appears to fix this

**Comment by kalyazin** [comment]
@antalk2, looks like it was my fault not adding a space after the first '-'. With the space it works in both cases.
Thanks.

**Comment by wasamasa** [comment]
@antalk2 Would you mind creating a pull request so that this patch can be made part of yaml-mode?

**Comment by wasamasa** [comment]
OK, fixed with #34.  Thanks again!
